tests/unit/test_data_interface.py

```diff
@@ -267,3 +282,59 @@ def testSearchReferences(self):
                 for repoReference, reference in utils.zipLists(
                         repoReferences, references):
                     self._assertEqual(repoReference, reference)
+
+    def testSearchRnaQuantificationSets(self):
+        self._testSearchMethodInContainer(
+            'getRnaQuantificationSets',
+            self._client.search_rna_quantification_sets,
+            self._repo.getDatasets())
+
+    def testSearchRnaQuantifications(self):
+        # TODO Can't use because of weird search_rna_quantifications signature
```

Comment:
@saupchurch the search_rna_quantifications signature has both dataset_id and rna_quantification_set_id as parameters... why not just the later since it contains the former?  That is, dataset_id seems redundant to me (we should change the schema if we delete it).